src/gtclang/Frontend/GTClangPreprocessorAction.cpp

```diff
@@ -550,7 +639,8 @@ class GTClangLexer {
   ///
   /// `#pragma gtclang CLAUSE_1 [, ... CLAUSE_N]`
   ///
-  /// Note that there might be cleaner ways of doing this but it is not clear if custom pragmas can
+  /// Note that there might be cleaner ways of doing this but it is not clear if custom pragmas
+  /// can
```

Comment:
line break again -- I'm guessing you run this line with `clang-format` which uses the default configuration (i.e column width of 80 and thus messed up some of the comments.